src/components/WizardPattern/WizardPattern.js

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
this'll sound silly, but sometimes yah have a space between the local imports vs global ones... I don't think it makes much of a difference either way, but would recommend consistency regardless

Reply:
I'll make sure it's consistent. I wasn't paying too much attention here